semigroupoids.cabal

```diff
@@ -153,26 +153,31 @@ library
     build-depends: ghc-prim
 
   if !impl(ghc >= 8.0)
-    build-depends: semigroups >= 0.16.2 && < 1
+    build-depends: semigroups >= 0.18.5 && < 1
 
   if flag(containers)
     build-depends: containers >= 0.3 && < 0.7
 
   if flag(contravariant)
-    build-depends: contravariant >= 0.2.0.1 && < 2
+    build-depends: contravariant >= 1.5 && < 2
 
   if flag(distributive)
     build-depends: distributive >= 0.2.2 && < 1
 
   if flag(comonad)
-    build-depends: comonad >= 4.2.6 && < 6
+    build-depends: comonad >= 5.0.3 && < 6
 
   if flag(tagged)
-    build-depends: tagged >= 0.8.5 && < 1
+    -- TODO: bump to 0.8.6.1
```

Comment:
Does this PR depend on ekmett/tagged#51 in order to work? (I was about to merge this until I saw this TODO, so in the future, in would be helpful to indicate things like this upfront.)